marge/app.py

```diff
@@ -137,6 +137,12 @@ def regexp(str_regex):
         action='store_true',
         help='Enable processing MRs in batches\n',
     )
+    parser.add_argument(
+        '--batch-branch-name',
+        type=str,
+        default=None,
```

Comment:
You could have this default to `'marge_bot_batch_merge_job'`. It would simplify things in `BatchMergeJob.__init__()`.

However it would break the config check you've added below, but there were no previous checks for the use of batch specific configs without batching being enable.

Reply:
Done.